icons/betterdiscord.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>BetterDiscord</title><path d="M17.181 7.251C17.056 2.695 13.494.857 8.903.857H0v9.207l5.693 5.13v-9.76h3.311c3.536 0 3.536 4.447 0 4.447H6.817v4.248h2.187c3.536 0 3.536 4.445 0 4.445H0v4.576h8.903c4.59 0 8.151-1.839 8.278-6.395 0-2.097-.574-3.663-1.583-4.752 1.009-1.088 1.583-2.655 1.583-4.752M15.381.85h-.989q.516.259.966.571a6.66 6.66 0 0 1 2.12 2.389c.514.988.792 2.136.828 3.412v9.565c-.034 1.277-.314 2.424-.828 3.412a6.65 6.65 0 0 1-2.12 2.389q-.447.309-.935.555h.934c4.459.025 8.643-2.207 8.643-7.321V8.347C24.024 3.132 19.841.85 15.381.85"/></svg>
```

Comment:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>BetterDiscord</title><path d="M14.393.861q.514.258.964.57a6.6 6.6 0 0 1 2.122 2.387c.513.987.792 2.133.828 3.409v9.556c-.035 1.275-.313 2.422-.828 3.408a6.6 6.6 0 0 1-2.122 2.387 8 8 0 0 1-.933.555h.933c4.46.024 8.643-2.205 8.643-7.315V8.352c.024-5.21-4.16-7.49-8.62-7.49zM0 .867v9.197l5.693 5.127V5.44h3.31c3.537 0 3.537 4.444 0 4.444H6.817v4.244h2.188c3.536 0 3.536 4.441 0 4.441H0v4.57h8.904c4.59 0 8.151-1.836 8.278-6.388 0-2.094-.574-3.66-1.584-4.748 1.01-1.087 1.584-2.652 1.584-4.746-.125-4.553-3.687-6.39-8.278-6.39z"/></svg>
```

Seeing some minor differences near some curves on the D half of the logo. This one is smaller and looks better to me :)

Reply:
that looks much better than before ;)